Assets/StreamingAssets/CSharp/SettingsMenuFunctions.cs

```diff
@@ -1093,6 +1093,19 @@ public override void CancelSetting()
     }
 }
 
+public class UIAnimationSpeedSlider : GenericSlider
+{
+    public override void ApplySetting()
+    {
+        base.ApplySetting();
+    }
+
+    public override void CancelSetting()
+    {
+        base.CancelSetting();
+    }
+}
+
```

Comment:
You don't need to have this :).  Just change the ClassName from `UIAnimationSpeedSlider` to `GenericSlider`